jwql/website/apps/jwql/templates/api_landing.html

```diff
@@ -0,0 +1,152 @@
+{% extends "base.html" %}
+
+{% block preamble %}
+
+	<title>API - JWQL</title>
+
+{% endblock %}
+
+{% block content %}
+
+    <main role="main" class="container">
+        <h2>Explore the JWQL API</h2><hr>
+        <p align="justify">
+            <ul>
+                <li><a href="#introduction">What is the JWQL API?</a></li>
+                <li><a href="#examples">Examples</a></li>
+                <li><a href="#api_list">List of Avaliable Services</a></li>
+                <li><a href="https://jwql.readthedocs.io/en/latest/index.html?highlight=api">Documentation</a></li>
+            </ul>
+        </p>
+        <br>
+        
+        <div id="introduction">
+            <h4>What is the JWQL API?</h4><hr>
+            <p align="justify">
+                JWQL allows users to access the data available through our programmatic Application Programming Interface (API). In short, an API
```

Comment:
This is a great analogy!  I'm stealing this whenever I have to describe what a programmatic API is to somebody 😂